k8s/aws/openshift-installer/create-openshift-cluster.yml

```diff
@@ -0,0 +1,144 @@
+          shell: ssh -o StrictHostKeyChecking=no -i openshift.ppk centos@{{ master_dns.stdout }} 'bash -s' < dependency.sh master
+          ignore_errors: yes
+
+        - name: Installing dependecies on Nodes
+          shell: ssh -o StrictHostKeyChecking=no -i openshift.ppk centos@{{item}} 'bash -s' < dependency.sh 
+          with_items: "{{ workers_dns.stdout_lines }}"
+
+        - name: SSH from master to each Nodes using DNS
+          shell: ssh -o StrictHostKeyChecking=no -i openshift.ppk centos@{{ master_dns.stdout }} 'sudo su -c "ssh -o StrictHostKeyChecking=no root@{{item}} ls"'
+          with_lines: cat ./dns.csv
+
+        - name: SSH from master to each Nodes using Public_IP
+          shell: ssh -o StrictHostKeyChecking=no -i openshift.ppk centos@{{ master_dns.stdout }} 'sudo su -c "ssh -o StrictHostKeyChecking=no root@{{item}} ls"'
+          with_lines: cat ./ip.csv  
+        
+        - name: Replacing master_dns name in inventory
+          replace:
+            path: ./inventory.yml
+            regexp: "master_pub_dns"
+            replace: "{{ master_dns.stdout }}"
+
+        - name: Replacing master_dns name in inventory
+          replace:
+            path: ./inventory.yml
+            regexp: "master_pub_dns"
+            replace: "{{ master_dns.stdout }}"  
+
+        - name: Get IP list
+          shell: cat ./ip.csv
+          register: ip_list
+
+        - name: Replace 
+          replace:
+            path: ./inventory.yml
+            regexp: "{{item.0}}"
+            replace: "{{item.1}}"
+          with_together: 
+            - [ 'master_pub_ip', 'node1_pub_ip', 'node2_pub_ip', 'node3_pub_ip' ]
+            - "{{ ip_list.stdout_lines }}"            
+
+        - name: Getting the content of inventory
+          shell: cat ./inventory.yml
+          register: inventory_spec
+
+        - name: Copying the content of inventory into master
+          args:
+            executable: /bin/bash
+          shell: scp -o StrictHostKeyChecking=no -i openshift.ppk inventory.yml centos@{{ master_dns.stdout }}:/home/centos/
+
+        - name: Running openshift pre-requisites
+          shell: ssh -o StrictHostKeyChecking=no -i openshift.ppk centos@{{ master_dns.stdout }} 'sudo su -c "ansible-playbook -i /home/centos/inventory.yml /home/centos/openshift-ansible/playbooks/prerequisites.yml -vv"'
+          async: 100
+          ignore_errors: yes
+
+        - name: Running openshift pre-requisites
+          shell: ssh -o StrictHostKeyChecking=no -i openshift.ppk centos@{{ master_dns.stdout }} 'sudo su -c "ansible-playbook -i /home/centos/inventory.yml /home/centos/openshift-ansible/playbooks/prerequisites.yml -vv"'
+
+        - name: Installing Openshift cluster
+          shell: ssh -o StrictHostKeyChecking=no -i openshift.ppk centos@{{ master_dns.stdout }} 'sudo su -c "ansible-playbook -i /home/centos/inventory.yml /home/centos/openshift-ansible/playbooks/deploy_cluster.yml -vv"'
+
```

Comment:
Give a comment why we execute the same task again

Reply:
@gprasath I have made the change.